module-system/module-implementations/sov-bank/src/call.rs

```diff
@@ -137,29 +137,41 @@ impl<C: sov_modules_api::Context> Bank<C> {
         Ok(CallResponse::default())
     }
 
-    /// Mints the `coins` set by the address `minter_address`. If the token address doesn't exist return an error.
+    /// Mints the `coins`to the address `mint_to_address` using `context.sender()` as the authorizer.
+    /// Returns an error if the token address doesn't exist or `context.sender()` is not authorized to mint tokens.
+    /// Calls the [`Token::mint`] function and update the `self.tokens` set to store the new balance.
+    pub fn mint_from_eoa(
```

Comment:
super minor nit, but it would be good to say what does "EOA" stands for.

Reply:
Done!